tests/RobotFramework/tests/cumulocity/registration/registration_livecycle.robot

```diff
@@ -73,6 +86,13 @@ Register service on a child device via MQTT
     # Check service registration
     Check Service    child_sn=${CHILD_XID}    service_sn=${CHILD_XID}:service:custom-app    service_name=custom-app    service_type=custom-type    service_status=up
 
+    # Deregister service on a child device
+    Execute Command    mosquitto_sub --remove-retained -W 3 -t 'te/device/${CHILD_SN}/+/+/#'    exp_exit_code=27
```

Comment:
```suggestion
    Execute Command    mosquitto_sub --remove-retained -W 3 -t 'te/device/${CHILD_SN}/service/custom-app/#'    exp_exit_code=27
```
That's the topic filter to target the given service. That topic filter `te/device/${CHILD_SN}/+/+/#` covers the device as well along with all of its services as well. If the intent was to deregister all the services of that given child device, then the topic filter should have been `te/device/${CHILD_SN}/service/+/#`.